util/write-package-info.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const path = require('path');
+
+const workingDirectory = process.argv[2];
+const destinationFile = process.argv[3];
+const packageJsonPath = path.join(process.cwd(), workingDirectory, 'package.json');
+const configDest = path.join(process.cwd(), workingDirectory, destinationFile);
+const packageJson = require(packageJsonPath);
+const packageInfo = {
+  name: packageJson.name,
+  version: packageJson.version
+};
+const output = 'export default ' + JSON.stringify(packageInfo, null, 2) + ';';
```

Comment:
As noted earlier, the trailing `+ ';'` will result in eslint issues in our Vue package. Can we make this conditional?

Reply:
I excluded this file using the eslintignore in the vue package

Reply:
Really? I'm seeing this after a clean install:
![screen shot 2018-03-19 at 9 23 04 am](https://user-images.githubusercontent.com/20212241/37608167-1a23a4e8-2b57-11e8-8faf-15a05a03a06f.png)